packages/form-field/src/FormField/FormField.tsx

```diff
@@ -47,6 +47,10 @@ export const FormField = forwardRef<HTMLDivElement, FormFieldProps>(
       optional,
       id,
       'data-lgid': dataLgId,
+      'aria-label': ariaLabelProp,
+      'aria-labelledby': ariaLabelledbyProp,
+      'aria-invalid': ariaInvalidProp,
+      readOnly,
```

Comment:
I think we should match everything thats returned with input props:

https://github.com/mongodb/leafygreen-ui/blob/c2e7029733f62c868bb369c7a5e66cac8fecb6ff/packages/form-field/src/FormField/useFormFieldProps.ts#L7-L14